dkms.8.in

```diff
@@ -939,6 +939,31 @@ Run no more than this number of jobs in parallel.
 .TP
 .B $compress_gzip_opts, $compress_xz_opts, $compress_zstd_opts
 Control how modules are compressed. By default, the highest available level of compression is used.
+.TP
+.B $post_transaction
+If the directive is set to any non null-value, the content of the directive will be executed as a command at the end of
```

Comment:
Is it a command as in `exec()` or command as in `sh -c`? I think the later. It might be good to know if `&&` and various shell constructs can be used.

Reply:
I still need to add the tests, so I will add something pertaining to this.